### DISCUSSION TITLE
Description for Fileds

### DISCUSSION THREAD

**Discussion by maximiliano-gt** [strapi/strapi#8008] [Legacy Topics]
Hello, i could not find a way to add some description to a field when creating a new collection type.
For example: I want to add some information next to the name of my image field. 

Image - (Max size ... / Resolution ... )

**Comment by derrickmehaffy** [comment]
Hi @maximiliano-gt 

You aren't able to do this during the initial model creation but if you go into the edit view on the right there is a small gear icon that says "Configure the view". From there you can select the field and add a description to it.
